### PR TITLE
Fb attributes

### PR DESCRIPTION
**Pull request by labkey-adam**

### CODE REVIEW

File: api/src/org/labkey/api/view/NavTree.java [hunk 1]
```diff
@@ -446,7 +447,7 @@ public String getScript()
         if (_usePost && null != _script)
             throw new IllegalStateException("Can't specify both usePost and setScript");
 
-        return _usePost ? PageFlowUtil.postOnClickJavaScript(_href) : _script;
+        return _usePost ? (null != _confirmMessage ? PageFlowUtil.confirmAndPostJavaScript(_confirmMessage, _href) : PageFlowUtil.postOnClickJavaScript(_href)) : _script;
```
Comment (ilikepies101) [inline api/src/org/labkey/api/view/NavTree.java]: This is a styling choice but personally nested ternaries are pretty hard to follow, so if we could change this to standard if statement formatting that would be great.

Comment (labkey-adam) [reply api/src/org/labkey/api/view/NavTree.java]: I added confirmMessage to the existing PageFlowUtil method instead of different methods for confirm vs. not. This simplifies a lot of the logic, so good suggestion.
